components/editor/modules/embed/index.js

```diff
@@ -186,6 +198,10 @@ const YOUTUBE_REGEX = /^http(?:s?):\/\/(?:www\.)?youtu(?:be\.com\/watch\?v=|\.be
 // One capturing group at match[1] that catches the video id
 const VIMEO_REGEX = /^(?:http|https)?:\/\/(?:www\.)?vimeo.com\/(?:channels\/(?:\w+\/)?|groups\/(?:[^/]*)\/videos\/|)(\d+)(?:|\/\?)$/
 
+const COMMENT_REGEX = new RegExp(
+  `^${FRONTEND_BASE_URL}/(?:(?:dialog\\?t=article&id=[a-f\\d-]+&)|(?:beitrag/diskussion\\?))focus=([a-f\\d-]+)$`
```

Comment:
Expression seems not to cover comments in «Allgemeines Feedback» come with `t=general`, like so:
- https://www.republik.ch/dialog?t=general&focus=b18a4684-2b4d-4df6-bab2-72905d06dfce

Reply:
It doesn't but it should. I'll fix.

Reply:
Dedicated discussion articles, like `/something-something/diskussion`, seem not to be matched:
- https://www.republik.ch/2021/04/26/wie-geht-es-ihnen-im-corona-limbo/diskussion?focus=96e268ba-8e35-4ac0-a1e0-73766eaacbe8